src/Libraries/CoreNodeModels/ColorPalette.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.Xml;
+using System.ComponentModel;
+using System.Linq;
+using System.Windows.Media;
+using ProtoCore.AST.AssociativeAST;
+using DSColor = DSCore.Color;
+using Autodesk.DesignScript.Runtime;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph;
+
+using CoreNodeModels.Properties;
+
+namespace ColorNode
+{
+    /// <summary>
+    ///     Custom Color UI node.
+    /// </summary>
+    [NodeName("Color Picker")]
+    [NodeDescription("ColorPaletteDescription", typeof(Resources))]
+    [NodeCategory("Core.Color.Create")]
+    [NodeSearchTags("ColorPaletteSearchTags", typeof(Resources))]
+    [IsDesignScriptCompatible]
+    [OutPortNames("Color")]
+    [OutPortTypes("Color")]
+    [OutPortDescriptions("Selected Color.")]
+    public class nColorPicker : NodeModel, INotifyPropertyChanged
+    {
+        private Color scolor = Color.FromArgb(255, 0, 0, 0);
```

Comment:
I think what you will need to do is essentially do all conversions from windows color types to DSCore.Color types on the view so that you do not require any of those references in the coreNodeModels project. The Model should only know about DSCore instead of other color types.